convert2rhel/actions/system_checks/convert2rhel_latest.py

```diff
@@ -60,6 +60,12 @@ def run(self):
 
         if not system_info.has_internet_access:
             logger.warning("Skipping the check because no internet connection has been detected.")
+            self.add_message(
+                level="WARNING",
+                id="CONVERT2RHEL_LATEST_CHECK_SKIP_NO_INTERNET",
+                title="Skipping convert2rhel latest version check",
+                description="Skipping the check because no internet connection has been detected.",
```

Comment:
Maybe this description should be moved to a variable to not mismatch at some point from the warning logger?

Reply:
Sounds good